tests/integration/dynamic_ip_test.py

```diff
@@ -1077,3 +1078,59 @@ def test_static_ip_with_routes_switch_back_to_dynamic(
         if route[RT.NEXT_HOP_INTERFACE] == DHCP_CLI_NIC
     ]
     assert not current_config_routes
+
+
+@pytest.fixture(scope="function")
+def eth1_with_dhcp6_no_dhcp_server():
+    # Cannot depend on eth1_up fixture as the reproducer requires the
+    # veth profile been created with DHCPv6 enabled.
+    logging.debug(libnmstate.show())
+    iface_state = {
+        Interface.NAME: "eth1",
+        Interface.TYPE: InterfaceType.ETHERNET,
+        Interface.STATE: InterfaceState.UP,
+    }
+    iface_state[Interface.IPV4] = create_ipv4_state(enabled=False)
+    iface_state[Interface.IPV6] = create_ipv6_state(
+        enabled=True, dhcp=True, autoconf=False
+    )
+    libnmstate.apply({Interface.KEY: [iface_state]})
+    try:
+        yield iface_state
+    finally:
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: "dummy00",
```

Comment:
this is dummy00 but originally the test uses eth0, it would be better to use a constant to avoid discrepancies.

Reply:
Thanks. Fixed.